content/books/bolthole-aces/chapter-01-Innocence Lost.md

```diff
@@ -0,0 +1,803 @@
+---
+title: "Chapter 01: Innocence Lost"
+date: 2018-06-17
+lastmod: 2018-06-17
+author: "P.A.B.Arianis (paladin3712x)"  
+
+---
+
+**3 years, 10 months, 3 weeks AV**  
```

Comment:
Make sure to add a double space at the end of a line if you want to have the very next line appear directly underneath.